### ISSUE TITLE
can you suggest the number of samples to get a decent accuracy

### ISSUE THREAD

**Issue by cogmeta** [friday27/KeywordSpotting#3]
What is required number of training examples to get a decent accuracy? Even with 500 training examples, the validation accuracy is bad. The training accuracy is around 90% but the validation accuracy keeps dropping below 50%. Any reasons for overfitting?

**Comment by XiaoyuGuo123** [comment]
I am a graduate student of Dalian university of technology in China. I want to complete my research on wake-up word recognition, but I don't have a wake-up word dataset. I would like to ask for your help, if it is convenient for you to send me the data set of your experiment, I would be very grateful.If it comes to issues like privacy or infringement, you don't have to worry about telling me directly that you can't.I am very glad to meet you.I hope you have a chance to visit China.My email address is 1179522397@qq.com.

**Comment by ucasiggcas** [comment]
@XiaoyuGuo123 
大佬有语料吗？
这个项目怎么开始啊？
多谢
